tests/JWT.Tests.Common/Algorithms/ECDSAAlgorithmTests.cs

```diff
@@ -57,7 +57,7 @@ public void Sign_Should_Throw_Exception_When_PrivateKey_Is_Null(Func<ECDsa, ECDS
         [DataTestMethod]
         public void Ctor_Should_Not_Throw_Exception_When_Certificate_Has_No_PrivateKey(Func<X509Certificate2, ECDSAAlgorithm> algFactory)
         {
-            var bytes = Encoding.ASCII.GetBytes(TestData.ServerEcdsaPublicKey);
+            var bytes = Convert.FromBase64String(TestData.ServerEcdsaPublicKey);
```

Comment:
I'm on linux so I get this issue:
https://github.com/dotnet/runtime/issues/47005

This code should work on all platforms.